web/templates/settings/profile.html

```diff
@@ -37,16 +37,20 @@ <h3 class="ui header">{{ .T "General" }}</h3>
                 required
                 disabled />
             </div>
-            <div class="field">
-              <label>
-                {{ .T "Alternative username (cannot be used for login)" }}
-              </label>
-              <input
-                type="text"
-                name="username_aka"
-                value="{{ $d.username_aka }}"
-                placeholder="{{ .T "Also known as..." }}" />
-            </div>
+            {{ if has .Context.User.Privileges 4 }}
```

Comment:
4 is not even close to correct

src: https://github.com/osuAkatsuki/score-service/blob/4a96152ac128df8f6c3c7595f7b920d86ec5e8a8/app/constants/privileges.py#L32

Reply:
My bad, I forgot Akatsuki has 2 different types of supporter 😅

Reply:
resolved in f320a29714e979c959f56e8c2124657b5f251123